### ISSUE TITLE
Shield Advanced: Measure latency to a domain on *.fr-stage.cloud.gov before & after DNS cutover

### ISSUE THREAD

**Issue by jameshochadel** [cloud-gov/product#2352]
We want to know if serving content over CloudFront will have an impact on latency. 

* We can't use Gorouter response time from logs because that's measured after requests have already passed through CloudFront & LBs. 
* The only CloudWatch metric available on ALBs is `TargetResponseTime`, which measures response time of the upstream target, not of the LB itself. 
* [ALB access logs](https://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-access-logs.html) include fields we could use, but we don't have a way to query them easily. (Fields are request_processing_time + target_processing_time + response_processing_time.)
* [CloudFront standard logs](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/AccessLogs.html#LogFileFormat), equivalent to ALB access logs, have the time-taken field, but logging isn't turned on for the CloudFront distribution right now. (I'll address that in a separate ticket.)

Options are:
* Turn on CloudFront logging and find a way to analyze the logs from CloudFront and the ALB. Possibly choose random samples from the access logs and just analyze them in Excel or similar.
* Run a load test against the domain. Requires figuring out how to set up the load test with non-interactive auth and some requests to make to the API server.

**Comment by pburkholder** [comment]
Did we find a way to measure before/after latency or impact? Or did we decide just not to do this?

**Comment by markdboyd** [comment]
@pburkholder I know we chatted offline, but currently, we have no tooling or methodology for doing this kind of traffic profiling effectively. So my feeling is that the work and effort involved in figuring out how to actually profile our traffic effectively is itself a PI epic level objective. And if we took on that work before implementing Shield Advanced, we'd never actually get to the point of implementing it.

Also, from speaking to other programs in TTS using Shield Advanced, no one has mentioned any latency as a possible concern
